backend/src/packages/chaiNNer_stable_diffusion/stable_diffusion/high_level/txt2img.py

```diff
@@ -0,0 +1,142 @@
+from __future__ import annotations
+
+from typing import Optional
+
+import cv2
+import numpy as np
+import torch
+
+from nodes.group import group
+from nodes.utils.utils import nearest_valid_size
+from nodes.impl.stable_diffusion import (
+    CLIPModel,
+    LatentImage,
+    Sampler,
+    Scheduler,
+    StableDiffusionModel,
+    VAEModel,
+)
+from nodes.properties import expression
+from nodes.properties.inputs import EnumInput, NumberInput, SliderInput, TextAreaInput
+from nodes.properties.inputs.stable_diffusion_inputs import (
+    CLIPModelInput,
+    StableDiffusionModelInput,
+    VAEModelInput,
+)
+from nodes.properties.outputs import ImageOutput
+
+from .. import abstract_group
+
+
+@abstract_group.register(
+    "chainner:stable_diffusion:txt2img",
+    name="Text to Image",
+    description="Use a diffusion model to generate an image based on a text prompt",
+    icon="MdAutoAwesome",
+    inputs=[
+        StableDiffusionModelInput(),
+        CLIPModelInput(input_type=expression.CLIPModel(arch_as="Input0")),
+        VAEModelInput(),
+        SliderInput(
+            "Width",
+            minimum=64,
+            default=512,
+            maximum=2048,
+            slider_step=64,
+            controls_step=64,
+        ),
+        SliderInput(
+            "Height",
+            minimum=64,
+            default=512,
+            maximum=2048,
+            slider_step=64,
+            controls_step=64,
+        ),
+        TextAreaInput("Prompt").make_optional(),
+        TextAreaInput("Negative Prompt").make_optional(),
+        group("seed")(NumberInput("Seed", minimum=0, default=42, maximum=4294967296)),
```

Comment:
We now have a seed input that gives you a seed object. See this example: https://github.com/chaiNNer-org/chaiNNer/blob/04ba6fe7454de042b0351c109c65aed4ef6fc32f/backend/src/packages/chaiNNer_standard/image_filter/noise/add_noise.py#L49

You can then use `seed.to_u32()` to get a number within limits. https://github.com/chaiNNer-org/chaiNNer/blob/04ba6fe7454de042b0351c109c65aed4ef6fc32f/backend/src/nodes/utils/seed.py#L28